### ISSUE TITLE
Auto-expanding of links (Jira, GitHub)

### ISSUE THREAD

**Issue by pneff** [6uhrmittag/taskbutler#158]
I just came across Taskbutler when looking for tools that would automatically expand links in my Todoist. Specifically I often add links to Jira tickets and GitHub PRs to process. Those I would like to change to have the title automatically.

At the moment I do this with a little Python script I have integrated with Alfred. But Taskbutler looks like a good basis to automate this further.

Before investing into this, I wanted to open this ticket to see if you would merge any such PRs if I contributed them to the repo.

**Comment by 6uhrmittag** [comment]
Thanks for reaching out! 
I'd be happy to merge such a feature.

Please use the branch [beta](https://github.com/6uhrmittag/taskbutler/tree/beta) to create a branch, as I'm just in the process of migrating from Travic-CI to GithubActions.

Depending on your implementation, it would probably good to limit the search&expand to specific projects in Todoist(configured as key in `config.ini`). But as long as it gets an `on/off` key in the `config.ini` I'm fine with any implementation that fits your needs.

**Comment by pneff** [comment]
Thank you @6uhrmittag. I noticed when looking at the code, that you already have some GitHub config options. Am I right in assuming that this was meant not for the link expansion I mention but more for syncing entire issue lists from GitHub to Todoist?

**Comment by 6uhrmittag** [comment]
> more for syncing entire issue lists from GitHub to Todoist?

Yes, this. I startet a 2-way sync, but never finished it(code in [e8bbd51](https://github.com/6uhrmittag/taskbutler/blob/e8bbd51c52d546683c5c2aefb356353e9a723c2d/taskbutler/taskbutler.py#L454)).

It had no auto-expand feature though, since it created the issues directly via API, it used the returned URL for linking to it.
